src/main/java/com/habityouheard/habityouheard/models/HabitMeta.java

```diff
@@ -8,20 +8,22 @@
 
 @Entity
 public class HabitMeta {
+
     @Id
     @GeneratedValue
     private int id;
 
+    private boolean completedHabit;
```

Comment:
Isn't this the same field as line 21?

Reply:
Fixed in commit 370e49a7bb5caa4a11feea91b05f146ccff72d87